internal/update/update.go

```diff
@@ -21,7 +21,7 @@ import (
 //go:embed update.rego
 var updateModule string
 
-const CheckAndWarnIgnoreVariable = "REGAL_FEATURES_REMOTE_CHECK_VERSION"
+const CheckVersionDisableEnvVar = "REGAL_FEATURES_REMOTE_CHECK_VERSION_DISABLE"
```

Comment:
I'd go with REGAL_DISABLE_VERSION_CHECK, but naming is hard :D

Reply:
Ok, let's go with that, it's shorter and reads more for humans than for computers.

My thinking with the other one was:

- it's for regal
- then it's for regal features
- then it's for remote features
- then it's for check-version
- then it's to disable it.

but it's perhaps verbose.

@srenatus had thoughts here too so will let him comment, but have used REGAL_DISABLE_VERSION_CHECK for now.